storm-core/src/jvm/backtype/storm/utils/Utils.java

```diff
@@ -503,6 +505,24 @@ public static boolean getBoolean(Object o, boolean defaultValue) {
         }
     }
 
+    public static String getString(Object o, String defaultValue) {
```

Comment:
Can we inline this or move it someplace else, it does not feel generic enough to be a part of Utils.